src/components/page/JoinNowB.svelte

```diff
@@ -0,0 +1,360 @@
+<script>
+  import { onMount } from "svelte";
+  import Arrow from "@components/images/Arrow.svelte";
+
+
+  onMount(async () => {
+    const input = document.querySelector('#first_name');
+    const form = document.querySelector('#join-form');
```

Comment:
There's a cleaner way to do this in Svelte... see https://svelte.dev/docs/element-directives#bind-this

add `let input;` and `let form;` outside of the onMount, and in the elements rather than the id, use `bind:this={input}` and `bind:this={form}`